### PR TITLE
refactor: remove Serializable bound in service interfaces

### LINKED ISSUE #29: Type parameters should not be bound by Serializable in service interfaces
https://github.com/AppJars/commons-backend/blob/cc07f0892e0ca613803036fb2d639e47ce522ce9/commons-business/src/main/java/com/appjars/saturn/service/CreationService.java#L32

etc.

### PR DESCRIPTION
**Pull request by javier-godoy**
Close #29

### CODE REVIEW

File: commons-data/src/main/java/com/appjars/saturn/dao/CreationDao.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 import java.io.Serializable;
```
Comment (mlopezFC) [inline commons-data/src/main/java/com/appjars/saturn/dao/CreationDao.java:22]: Remove also this import

File: commons-data-impl/src/main/java/com/appjars/saturn/dao/jpa/ConversionJpaDaoSupport.java [hunk 2]
```diff
@@ -22,7 +22,6 @@
 import java.io.Serializable;
```
Comment (mlopezFC) [inline commons-data-impl/src/main/java/com/appjars/saturn/dao/jpa/ConversionJpaDaoSupport.java:22]: Why this one was not removed?

**Review by mlopezFC — CHANGES_REQUESTED** [review]
Just a couple of small things.

File: commons-data-impl/src/main/java/com/appjars/saturn/dao/jpa/ConversionJpaDaoSupport.java (hunk 2 above)
Comment (javier-godoy) [reply commons-data-impl/src/main/java/com/appjars/saturn/dao/jpa/ConversionJpaDaoSupport.java:22]: Section 2.4 of JSR-317 (Primary Keys and Entity Identity) specifies that 

> The following rules apply for composite primary keys:
> (...) The primary key class must be serializable.

Additionally:
> A simple primary key or a field or property of a composite primary key should be one of the following types: any Java primitive type; any primitive wrapper type; java.lang.String; java.util.Date; java.sql.Date; java.math.BigDecimal; java.math.BigInteger. 

Therefore, the fact that the key must be serializable is a constraint introduced by JPA (but it doesn't carry on to the presentation/service layers nor other persistence implementations). Implementations of `ConversionJpaDaoSupport` (or `JpaDaoSupport`) will have a key with an upper bound of `Serializable`, while the upper bound will be `Object` elsewhere (unless another non-JPA constraint apply)

File: commons-data/src/main/java/com/appjars/saturn/dao/CreationDao.java (hunk 1 above)
Comment (javier-godoy) [reply commons-data/src/main/java/com/appjars/saturn/dao/CreationDao.java:22]: Good catch.

File: commons-data-impl/src/main/java/com/appjars/saturn/dao/jpa/ConversionJpaDaoSupport.java (hunk 2 above)
Comment (mlopezFC) [reply commons-data-impl/src/main/java/com/appjars/saturn/dao/jpa/ConversionJpaDaoSupport.java:22]: Excellent, solved then

**Review by mlopezFC — APPROVED** [review]
Now it LGTM

**Pull request merged**